ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp

```diff
@@ -137,6 +137,21 @@ struct ExecuteBinaryBackwardBiasGelu {
 
 };
 
+struct ExecuteUnaryBackwardComparison {
```

Comment:
Does not look unary to me

Reply:
Should be renamed I think

Reply:
We have renamed it in the next PR whoch will be merged along with this

Reply:
Sure